Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -329,7 +330,7 @@ def set_preprocessor(self, preproc):
     def handleNewSignals(self):
         """Reimplemented from OWWidget.handleNewSignals."""
         self._update_class_selection()
-        self.apply()
+        self.commit()
```

Comment:
I suppose it would be better to call `self.unconditional_commit`. We probably want to force apply here.